articles/personal_growth.md

```diff
@@ -0,0 +1,33 @@
+---
+authorName: Sabrina Lange
+authorGithubUsername: sweinzie 
+issue: 97
+title: Personal Growth - Why every Project Should Have Gardeners
+---
+# {{page.title}}
+
+Personal Growth - Why every Project Should Have Gardeners
+
+Everyone started small. Even the tallest and strongest tree started as a seed and over time, thanks to a lot of sunlight, rainfall and good soil it became a deep-rooted, reliable and sturdy tree. And every year in the summertime it offers shade and fruits to anyone who appreciates it.
```

Comment:
Kürzungspotential: "And every year in the summertime it offers shade and fruits to anyone who appreciates it." ganz weglassen